examples/js/controls/VRControls.js

```diff
@@ -9,22 +9,28 @@ THREE.VRControls = function ( object, onError ) {
 
 	var vrInputs = [];
 
-	function filterInvalidDevices( devices ) {
+function filterInvalidDevices( devices ) {
 
 		// Exclude Cardboard position sensor if Oculus exists.
 
 		var oculusDevices = devices.filter( function ( device ) {
-
-			return device.deviceName.toLowerCase().indexOf( 'oculus' ) !== - 1;
+			if(device.deviceName) {
+				return device.deviceName.toLowerCase().indexOf( 'oculus' ) !== - 1;
+			} else {
+				return [];
```

Comment:
I think the function passed to Array.filter() is supposed to return a boolean.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter